### PR TITLE
DOCS-12133-added-classifier-values

### PR DESCRIPTION
**Pull request by luanamulesoft**

### CODE REVIEW

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 1]
```diff
@@ -1269,7 +1268,35 @@ Supported Values:
...
+* SOAP API
+** `wsdl` (with `zip`, `wsld` or `xml` as packaging)
+* Custom
+** `custom`
+* Application
+** `mule-application` (with `jar` as packaging)
+* Policy
+** `mule-policy` (with `jar` as packaging) + `policy-definition` (with `yaml` as packaging)
+* Example
+** `mule-application-exampl`' (with `jar` as packaging)
```
Comment (LucasLudueno) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
** `mule-application-example`' (with `jar` as packaging)
```

**Review by LucasLudueno — DISMISSED** [review]

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 2]
```diff
@@ -1269,7 +1268,35 @@ Supported Values:
 * `policy`
 * `app`
 * `extension`
-* `external-library` |
+* `external-library`
+
+If it is uploaded, the type is inferred from the classifier of the file.
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc:1273]: Please clarify to what "it" refers: the type? the classifier? Something else?

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 3]
```diff
@@ -1269,7 +1268,35 @@ Supported Values:
...
+
+Depending on the type of asset, some possible classifier values are:
+
+* REST API
+** `oas`  (with `zip`, `yaml` or `json` as packaging)
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]: Oxford comma:
```suggestion
** `oas`  (with `zip`, `yaml`, or `json` as packaging)
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 4]
```diff
@@ -1269,7 +1268,35 @@ Supported Values:
...
+** `raml` (with `zip` or `raml` as packaging)
+* RAML Fragment
+** `raml-fragment` (with `zip` or `raml` as packaging)
+* SOAP API
+** `wsdl` (with `zip`, `wsld` or `xml` as packaging)
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
** `wsdl` (with `zip`, `wsld`, or `xml` as packaging)
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 5]
```diff
@@ -1269,7 +1268,35 @@ Supported Values:
...
+* Custom
+** `custom`
+* Application
+** `mule-application` (with `jar` as packaging)
+* Policy
+** `mule-policy` (with `jar` as packaging) + `policy-definition` (with `yaml` as packaging)
+* Example
+** `mule-application-example`' (with `jar` as packaging)
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]: extra character?
```suggestion
** `mule-application-example` (with `jar` as packaging)
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 6]
```diff
@@ -1298,6 +1325,44 @@ Supported Values:
...
 
+=== exchange asset mutableDataUpload
+
+----
+> exchange asset mutableDataUpload [properties] <assetIdentifier>
+----
+
+This command modifies the mutable data of an already created asset, including tags, categories, fields, and documentation.
+
+Use the following format for the _assetIdentifier_ argument:
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
Use the following format for the `_assetIdentifier_` argument:
```

File: modules/ROOT/pages/anypoint-platform-cli-commands.adoc [hunk 7]
```diff
@@ -1298,6 +1325,44 @@ Supported Values:
...
+
+`[_<groupID>_]/_<assetID>_/_<version>_`
+
+If _group_id_ is not specified, it defaults to the currently selected organization ID.
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-platform-cli-commands.adoc]:
```suggestion
If `_group_id_ `is not specified, the value for this argument defaults to the currently selected organization ID.
```

**Review by kbostrom — DISMISSED** [review]
PR with comments

**Review by kbostrom — APPROVED** [review]
PR CE RA

**Pull request merged**